pkg/cli/mirror/catalog_images.go

```diff
@@ -25,6 +31,23 @@ import (
 	"github.com/openshift/oc-mirror/pkg/image/builder"
 )
 
+const (
+	opmCachePrefix           = "/tmp/cache"
+	opmBinarySuffix          = "opm"
+	opmBinaryDir             = "usr/bin/registry"
+	cacheLocationPlaceholder = "cacheLocation.txt"
```

Comment:
Seems like this should go into pkg/config/metadata.go as a constant and update the "tree" diagram in that file

Reply:
Done, good catch